tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def change_test_dir(request, monkeypatch):
+    monkeypatch.chdir(request.fspath.dirname)
```

Comment:
magst du hie rnoch einen wieso-weshalb-warum kommentar ergänzen?

Reply:
ah sehr guter Punkt.
done in cc27080155f5e5a1a69ac8d963ddbe0b0e0bc146 and f47c922f30e93c16cfee65642cd7a799a98d5260